metadata.rb

```diff
@@ -7,6 +7,7 @@
 version          '1.4.0'
 source_url       'https://github.com/sous-chefs/fileutils'
 issues_url       'https://github.com/sous-chefs/fileutils/issues'
+chef_version     '>=15.3'
```

Comment:
You have a duplicate entry for `chef_version` on line 2 that needs to be removed please.

Reply:
Chef version is fixed.

Reply:
You still need to add a space between `>=` and `15.3` please.